qa/suites/rados/basic/tasks/rados_api_tests.yaml

```diff
@@ -16,6 +16,9 @@ overrides:
         debug ms: 1
       mon:
         mon warn on pool no app: false
+      osd:
+        osd class load list: *
```

Comment:
this is not a valid YAML.  as `*` is used to start an anchor name. see https://yaml.org/spec/1.2/spec.html#*%20alias//

Reply:
might want to quote `*` like
```yaml
osd:
  osd class load list: "*"
```